scripts/verify-artifact-contents

```diff
@@ -19,13 +19,15 @@ jars_in_plugin_zip=$(zipinfo -1 "$plugin_zip" | grep -Po '(?<=git-machete-intell
 
 gradle_projects=$(cat settings.gradle | grep -Po "(?<=^include ').*(?=')" | sort)
 for project in $gradle_projects; do
-  if [[ -d $project/src/main ]]; then
-    if ! grep -qx "$project.jar" <<< "$jars_in_plugin_zip"; then
-      fail "$project.jar is expected but missing from $plugin_zip"
+  project_path=${project//:/\/}
```

Comment:
Another consequence of #833... this script is a somewhat primitive test to make sure that we've got Gradle properly configured (checks that all expected subprojects and some of the expected libraries are actually packaged)